root/scripts/searchpage.js

```diff
@@ -242,14 +251,16 @@ function fillSearchPage(searchResults) {
     }
 
     resultsFound = resultsCounter;
-    document.getElementById('searchHeader').innerHTML = `${resultsFound} ${filterTags.length !== 0 ? filterMessage : ''} recipes found for ${searchQuery}, page 1 of results`;
+    document.getElementById('searchHeader').innerHTML = `${resultsFound} ${filterTags.length !== 0 ? filterMessage : ''} recipes found for ${searchQuery}${ingredsIncluded !== null ? includeMessage : ''}${ingredsExcluded !== null ? excludeMessage : ''}, page 1 of results`;
   }
 }
 
 async function init() {
   const queryString = window.location.search;
-
   const searchParams = new URLSearchParams(queryString);
+  ingredsIncluded = searchParams.get('IngIncl');
+  ingredsExcluded = searchParams.get('IngExcl');
+
```

Comment:
Quick note, maybe visually it would be better to put these two text inputs under a [collapsible element](https://www.w3schools.com/howto/howto_js_collapsible.asp), a dropdown called "Ingredients -" would suffice. That way the user isn't overwhelmed with 3 text inputs for the search page.